modin/core/execution/dask/implementations/pandas_on_dask/partitioning/virtual_partition.py

```diff
@@ -485,7 +497,7 @@ class PandasOnDaskDataframeRowPartition(PandasOnDaskDataframeVirtualPartition):
 
     Parameters
     ----------
-    list_of_blocks : list
+    list_of_partitions : list
```

Comment:
```suggestion
    list_of_partitions : Union[list, PandasOnDaskDataframePartition]
```

To make consistent with parent-class. Possible, we also should change parameter description.

Reply:
The same as in https://github.com/modin-project/modin/pull/4695#discussion_r933324726

Reply:
Done.